compress.c

```diff
@@ -495,13 +495,11 @@ int comp_ac_add(struct Account *a, struct Mailbox *m)
  * Then determine the type of the mailbox so we can delegate the handling of
  * messages.
  */
-static int comp_mbox_open(struct Context *ctx)
+static int comp_mbox_open(struct Mailbox *m, struct Context *ctx)
 {
   if (!ctx || !ctx->mailbox || (ctx->mailbox->magic != MUTT_COMPRESSED))
```

Comment:
Since `mx_mbox_open(...)` includes `NULL` checking and initialises our the `Context`, is this guard statement still necessary?